roles/nginx/molecule/resources/inventory/group_vars/all.yml

```diff
@@ -0,0 +1,7 @@
+---
+nginx_conf_template: nginx_reverse_proxy_as_default.j2
```

Comment:
Why not `nginx_reverse_proxy.j2`?

Reply:
it's the same issue @paddyroddy was having in https://github.com/UCL-MIRSG/ansible-collection-infra/pull/44 in `nginx_reverse_proxy.j2` - there is an additional default server, so any traffic to hosts that do not match `nginx_server_name` are routed to this default server. So in the verify task for this role:
https://github.com/UCL-MIRSG/ansible-collection-infra/blob/b8f4dd174716cbb3652b31dba49fff4350d2c0e0/roles/nginx/molecule/resources/verify.yml#L3-L11
the host passed to nginx will be `localhost`, and as this doesn't match `nginx_server_name` the request will be sent to the default server (which returns nothing). In the test we could set the correct host through a header (`-H 'Host: {{.nginx_server_name  }}'`), and that would work fine. However, I don't think it's possible to do this in a web browser, so if we run the converge step locally we wouldn't be able to access the web UI. To get around this, @paddy is currently setting the hostname for the webserver to be `localhost`